jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/sync/SyncServerlessApplicationDialog.kt

```diff
@@ -118,21 +117,31 @@ class SyncServerlessApplicationDialog(
 
     private val parametersField = KeyValueTextField(message("serverless.application.sync.template.parameters"))
     private val tagsField = KeyValueTextField(message("tags.title"))
-    private val capabilitiesSelector = CapabilitiesEnumCheckBoxes()
-
     private var templateFileParameters = CloudFormationTemplate.parse(project, templateFile).parameters().toList()
     private val module = ModuleUtilCore.findModuleForFile(templateFile, project)
     private val settings: SyncSettings? = module?.let { SyncSettings.getInstance(it) }
     private val samPath: String = module?.let { relativeSamPath(it, templateFile) } ?: templateFile.name
     private val templateFunctions = SamTemplateUtils.findFunctionsFromTemplate(project, templateFile)
     private val hasImageFunctions: Boolean = templateFunctions.any { (it as? SamFunction)?.packageType() == PackageType.IMAGE }
+    private val activeStacks = runBlocking(getCoroutineBgContext()) {
+        project.getResourceNow(CloudFormationResources.ACTIVE_STACKS, forceFetch = true, useStale = false)
+    }
```

Comment:
since the val is being initialized as part of the dialog, won't this still block the UI thread?